crates/common/types/transaction.rs

```diff
@@ -35,6 +37,116 @@ pub enum Transaction {
     PrivilegedL2Transaction(PrivilegedL2Transaction),
 }
 
+/// The same as a Transaction enum, only that blob transactions are in wrapped format, including
+/// the blobs bundle.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum P2PTransaction {
```

Comment:
Why exactly do we need a new transaction type? What does this provide that the current transaction type does not? It makes me raise an eyebrow